sbtc/src/deposits.rs

```diff
@@ -262,10 +262,12 @@ impl DepositScriptInputs {
     /// the format for a principal from SIP-005. So the expected wire
     /// format is:
     ///
+    /// ```text
```

Comment:
What made you add this? Is there a docs command I should be running?

Reply:
I use an extension for formatting my comments and it will format the diagram below unless it knows that it shouldn't be. This is one way to tell it that these lines shouldn't be formatted.